site/docs/patterns/navigation.mdx

```diff
@@ -0,0 +1,179 @@
+    { src: "/img/patterns/navigation/multi_level_toggle_on.png", alt: "Example showing a multi level navigation expanded" },
+  ]}
+  label="Expand multi-level-group"
+/>
+
+#### Group overview page
+Clicking a parent item will not navigate the user but expand the group. If an overview page is required it should be the first in the group. Overview pages will not include a chevron on the right-hand side.
+
+<Diagram
+  src="/img/patterns/navigation/group_overview_page_toggle_on.png"
+  alt="Example showing group overview"
+  border
+/>
+
+## Combined navigation
+Combined navigation can be useful in website or application contexts when you want to provide users with a versatile and efficient way to navigate complex content or features. It's ideal when your experience comprises of multiple sub-apps, pages or sections which in turn also require their own navigation.
+
+<ImageSwitcher
+  images={[
+    { src: "/img/patterns/navigation/combined_navigation_example.png", alt: "Example showing a combined navigation" },
+    { src: "/img/patterns/navigation/combined_navigation_toggle_on.png", alt: "Example showing a combined navigation with icons" },
+  ]}
+  label="With icons"
+/>
+
+## Navigation layout
+
+#### Fixed navigation pane
+Typically, the vertical navigation remains fixed in width when the user resizes the browser. You can set it to the width of your choice but you should consider the overall label length, theme, density, and the screen size you are designing for when making your decision.
+
+When your application is responsive, the content area should use its own layout grid. As the user adjusts their browser width, the navigation will remain fixed until reaching a specified breakpoint, with only the content area adapting responsively based on the viewport.
+
+<Diagram
+  src="/img/patterns/navigation/fixed_navigation_pane.png"
+  alt="Example showing a fixed navigation pane"
+  border
+/>
+
+#### Responsive navigation pane
+The vertical navigation component can be configured for fully responsive application experiences. Use the underlying layout grid to set a minimum width for your navigation pane and adjust your content area based on your predefined breakpoints.
+
+<ImageSwitcher
+  images={[
+    { src: "/img/patterns/navigation/responsive_navigation_pane.png", alt: "Example showing a responsive navigation" },
+    { src: "/img/patterns/navigation/responsive_navigation_pane_toggle_on.png", alt: "Example showing a responsive navigation with grid" },
+  ]}
+  label="Show underlying grid"
+/>
+
+<ExampleContainer type="positive">
+  - Use a layout grid for the full width of your app’s resolution if it is not responsive, or just for the content area if it is.
+  - Use [Salt breakpoints](/salt/foundations/responsiveness#breakpoints) to determine how the navigation will display at various viewport sizes.
+</ExampleContainer>
+
+## Expanding the pattern
```

Comment:
Right hand nav is a bit awkward on the last heading, not sure why indented

Reply:
might be from the previous build, it was a ### and i just fixed it

Reply:
but I'll keep an eye to make sure it's fixed

Reply:
fixed :)